packages/website/components/account/filesManager/fileRowItem.js

```diff
@@ -1,242 +1,244 @@
-import clsx from 'clsx';
-import { useMemo, useRef } from 'react';
-import { renderToString } from 'react-dom/server';
-import { BsFillInfoCircleFill } from 'react-icons/bs';
+// import clsx from 'clsx';
```

Comment:
Can we get rid of this one all together?

Reply:
Yep! Need to keep the CSS File since it's still used for the Pins table. Added a TODO at the top of that file..